torchaudio/models/wavernn.py

```diff
@@ -324,3 +346,24 @@ def forward(self, waveform: Tensor, specgram: Tensor) -> Tensor:
 
         # bring back channel dimension
         return x.unsqueeze(1)
+
+
+def wavernn(checkpoint_name: str) -> WaveRNN:
+    r"""Get pretrained WaveRNN model.
+
+    Args:
+        checkpoint_name (str): The name of the checkpoint to load. Available checkpoints:
+
+            - wavernn_10k_epochs_8bits_ljspeech:
+
+                WaveRNN model trained with 10k epochs and 8 bits depth waveform on the LJSpeech dataset.
+                The model is trained using the default parameters and code of the examples/pipeline_wavernn/main.py.
+    """
+    if checkpoint_name not in _MODEL_CONFIG_AND_URLS:
+        raise ValueError("The checkpoint_name `{}` is not supported.".format(checkpoint_name))
```

Comment:
When validating a value and there is a (small number of) finite set of valid values, listing them out is more user-friendly.

Imagine that I tried to pass `wavernn_10k_epochs_8bits_ljspeech` but misspelled `wavernn_10k_epochs_8bits_ljspeeck`. If the error message only tells me it's invalid, then I have to search for the documentation to see what is correct. If the error message also tells what are the valid choices, then I can copy-paste the valid ones from the error message and retry at instant.

`not supported` is correct but sounds like it's planned to be supported, and I think `unexpected` is more regularly used.

`str.format` method is fine, but typically, f-string is more readable and the code becomes shorter.

```suggestion
        raise ValueError(
            f"Unexpected checkpoint_name: '{checkpoint_name}'. "
            f"Valid choices are; {list(_MODEL_CONFIG_AND_URLS.keys())}")
```

Reply:
Thanks for pointing it out. These are definitely better designs.
I've fixed them [here](https://github.com/pytorch/audio/pull/1612/files#diff-7f4ad259da182d3840fe4c75dd0c0b7b9fe30f31503df0eff87b1e5c9378cac4R363).